src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/ReserveVisitSlotDto.kt

```diff
@@ -20,9 +20,8 @@ data class ReserveVisitSlotDto(
   @Schema(description = "Prison Id", example = "MDI", required = true)
   @field:NotBlank
   val prisonCode: String,
-  @Schema(description = "Visit Room", example = "A1", required = true)
-  @field:NotBlank
-  val visitRoom: String,
+  @Schema(description = "Capacity group", example = "Main Group", required = false)
```

Comment:
So, looks like we're repurposing the `visitRoom` as `capacityGroup`. Wondering if we actually need to pass this to/from front-end at all (e.g. like here when reserving a slot)? We don't do anything with `visitRoom` in the front-end. Are there requirements as part of this for us to be doing something with `capacityGroup`?

Reply:
As discussed - we do in fact need it!